### DISCUSSION TITLE
Is it correct to set notifications like this?

### DISCUSSION THREAD

**Discussion by belhyun** [upptime/upptime#120] [Q&A] [answered]
I added the following to the repository token to set up email notification. Is there anything else I need to set up? Also, if you want to send an email to multiple people, can you set NOTIFICATION_EMAIL_TO to connect the email with,?

![image](https://user-images.githubusercontent.com/895026/102303309-40d4c280-3f9e-11eb-845e-8cf959d59cb2.png)

**Comment by AnandChowdhary** [comment, marked as answer]
This is correct, yes. Can you confirm if the emails are **not** going?

**Comment by AnandChowdhary** [comment]
> Also, if you want to send an email to multiple people, can you set NOTIFICATION_EMAIL_TO to connect the email with?

At this point, we don't (officially) support sending emails to multiple people, but because we use Nodemailer under the hood, chances are that a comma-separated list of emails will work.
